tests/bq_test_helpers.py

```diff
@@ -0,0 +1,96 @@
+"""
+Shared helper functions for BigQuery-related tests
+"""
+import datetime
+import warnings
+from collections import OrderedDict
+from typing import Any, List, Union, Dict
+
+import mock
+from google.cloud import bigquery
+from google.cloud.bigquery import TableReference
+from google.cloud.bigquery.table import TableListItem
+
+_TYPE_TO_FIELD_TYPE = {
+    str(int): 'INT64',
+    str(str): 'STRING',
+    str(float): 'FLOAT64',
+    str(datetime.date): 'DATE',
+    str(datetime.datetime): 'DATETIME'
```

Comment:
Should add TIMESTAMP to this list.

Reply:
Hmm yeah I think I meant to use `TIMESTAMP` rather than `DATETIME` here. (A quick scan on a recent combined dataset shows 0 `DATETIME` fields vs 32 `TIMESTAMP` fields in our tables.) Let me know if you had something else in mind.